### ISSUE TITLE
Arch: Speed up install

### ISSUE THREAD

**Issue by n0v1c3** [n0v1c3/linux#27]
Remove unnecessary packages from the installer and add them to a separate script. This new script can be run after installation is completed, this way they can be run in parallel with the rest of the OS.

**Comment by n0v1c3** [comment]
This will also help to break down the installation script into smaller more maintainable scripts.

**Comment by n0v1c3** [comment]
User adding can also be moved into a separate script. This way any required users can be added properly as well as the help breaking down the larger script.

**Comment by n0v1c3** [comment]
Breaking down into smaller more maintainable scripts will help with testing as well. I have currently been required to completely reinstall arch each time I am required to test a new feature.

**Testing of `arch-chroot` after system has been installed will be required for certain commands this will also apply to `cp` commands that have `/mnt/` as the root directory**

**Comment by n0v1c3** [comment]
`install/linux.sh` has been modified to install i3 and seems to be working. This issue will be closed however, a new issue may be created to add a "post-install" script that can be automatically run on next boot, this script will run the "install / config" of the non-essential software in the background.
